### PR TITLE
upload skipped resources data to elasticsearch

### PR DESCRIPTION
**Pull request by athiruma**
1. Upload data of "NOT_DELETE" or "SKIP" tag resources
2. Added extra argument to mail_message i.e "message_type=notification" or "message_type=delete" to all mail sending policies
3. Calculate ebs cost by using AWS pricing service

### CODE REVIEW

File: cloud_governance/policy/aws/skipped_resources.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+from cloud_governance.aws.zombie_non_cluster.run_zombie_non_cluster_policies import NonClusterZombiePolicy
+
+
+class SkippedResources(NonClusterZombiePolicy):
+
```
Comment (ebattat) [inline cloud_governance/policy/aws/skipped_resources.py:5]: Missing documentation

File: cloud_governance/common/clouds/aws/price/price.py [hunk 2]
```diff
@@ -50,3 +61,53 @@ def get_region_name(self, region_code):
...
+            ec2_lanuch_time = item_data['LaunchTime']
+            d1 = datetime.strptime(ec2_lanuch_time, "%Y-%m-%dT%H:%M:%S+00:00")
+            d2 = datetime.strptime(strftime("%Y-%m-%dT%H:%M:%S+00:00"), "%Y-%m-%dT%H:%M:%S+00:00")
+            diff = d2 - d1
+            diff_in_hours = diff.total_seconds() / 3600
+            ec2_cost = round(float(ec2_type_cost) * diff_in_hours, 3)
+            return round(ec2_cost, 3)
+        elif resource == 'ec2' and item_data['State']['Name'] != 'running':
+            return '0'
+        elif resource == 'ebs':
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/price/price.py:105]: Why did you keep EBS ?

File: cloud_governance/common/clouds/aws/price/price.py [hunk 3]
```diff
@@ -50,3 +61,53 @@ def get_region_name(self, region_code):
...
+            ebs_monthly_cost = '0'
+            if item_data['VolumeType'] == 'gp2':
+                ebs_monthly_cost = 0.1 * item_data['Size']
+            elif item_data['VolumeType'] == 'io1':
+                ebs_monthly_cost = 0.125 * item_data['Size']
+            else:
+                ebs_monthly_cost = 0.1 * item_data['Size']
+            return round(ebs_monthly_cost, 3)
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/price/price.py:113]: Need to change hard coded value

File: cloud_governance/common/clouds/aws/price/price.py [hunk 4]
```diff
@@ -50,3 +61,53 @@ def get_region_name(self, region_code):
...
+            'standard': 'Magnetic',
+            'sc1': 'Cold HDD'
+        }
+        key = ''
+        for volume_key, value in volume_types.items():
+            if volume_key in volume_type:
+                key = value
+        return self.get_price(region=self.get_region_name(region), volume_type=key, aws_resource='ebs')
+
+    def get_resource_cost(self, resource: str, item_data: dict):
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/price/price.py:83]: You should change resource to ec2?

File: cloud_governance/common/clouds/aws/price/price.py (hunk 2 above)
Comment (athiruma) [reply cloud_governance/common/clouds/aws/price/price.py:109]: if I change here, It leads to making lot changes.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**